CPPLINT.cfg

```diff
@@ -13,6 +13,9 @@ filter=-build/namespaces_literals
 filter=-build/include_subdir
 filter=-legal/copyright
 
+# Allow runtime references
+filter=-runtime/references
```

Comment:
I changed cpplint's config to allow runtime references. A reference is safer than a pointer in this instance.